### PR TITLE
[8.0][partner_contact_department] Add departments to contacts

### PR DESCRIPTION
**Pull request by yajo**
Reopening https://github.com/OCA/partner-contact/pull/228 and https://github.com/OCA/department/pull/51.

This module extends the functionality of the address book to support departments.

Department is a drop-down field in partner forms, and it refers to contact department in its own company.

@JavierIniesta @rafaelbn @dreispt @pedrobaeza @antespi

### CODE REVIEW

**Comment by rafaelbn** [comment]
Tested in runbot with base database, http://3137766-229-4a1c1d.runbot2.odoo-community.org

:+1: thanks!

Please @JavierIniesta review, thanks!

File: partner_contact_department/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+To configure departments, you need to:
+
+* Go to *Sales > Configuration > Address Book > Departments*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
```
Comment (pedrobaeza) [inline partner_contact_department/README.rst:27]: Explain in which tab do you find the department

**Comment by pedrobaeza** [comment]
Little nitpick, but :+1:

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Comment by JMODaERP** [comment]
I test in runbot :+1:

File: partner_contact_department/README.rst (hunk 1 above)
Comment (rafaelbn) [inline partner_contact_department/README.rst:27]: It's too obvious to explain this

Comment (pedrobaeza) [inline partner_contact_department/README.rst:27]: Maybe not. You are the one that always ask for detailed explanations. Saying "You'll find the field below Job position" is enough.

Comment (yajo) [inline partner_contact_department/README.rst:27]: Fixed.

**Comment by yajo** [comment]
3 x :+1: + nitpick fixed = LGTM please! :blush:

**Pull request merged**